ext/nnstreamer/tensor_converter/tensor_converter_protobuf.cc

```diff
@@ -0,0 +1,163 @@
+/* SPDX-License-Identifier: LGPL-2.1-only */
+/**
+ * GStreamer / NNStreamer tensor_converter subplugin, "protobuf"
+ * Copyright (C) 2020 Gichan Jang <gichan2.jang@samsung.com>
+ */
+ /**
+ * @file        tensor_converter_protobuf.cc
+ * @date        2 June 2020
+ * @brief       NNStreamer tensor-converter subplugin, "protobuf",
+ *              which converts protobuf byte stream to tensors.
+ * @see         https://github.com/nnsuite/nnstreamer
+ * @author      Gichan Jang <gichan2.jang@samsung.com>
+ * @bug         No known bugs except for NYI items
+ *
+ */
+
+ /**
+  * Install protobuf
+  * We assume that you use Ubuntu linux distribution.
+  * You may simply download binary packages from PPA
+  *
+  * $ sudo apt-add-repository ppa:nnstreamer
+  * $ sudo apt update
+  * $ sudo apt install libprotobuf-dev libprotobuf-lite17 libprotobuf17 protobuf-compiler17
+  */
+
+#include <iostream>
+#include <fstream>
+#include <typeinfo>
+#include <glib.h>
+#include <gst/gstinfo.h>
+#include <nnstreamer_plugin_api.h>
+#include <string>
+#include "nnstreamer_plugin_api_converter.h"
+#include <nnstreamer_log.h>
+#include "nnstreamer.pb.h"      /* Generated by `protoc` */
+
+namespace nnstreamer {
+namespace protobuf {
+
+  void init_pbc (void) __attribute__ ((constructor));
```

Comment:
I think you don't need to add an indent inside namespace

Reply:
Thank you! I will update it.

Reply:
I think this issue was not resolved yet :(